File: 0268-missing-number/0268-missing-number.cpp
class Solution {
public:
    int missingNumber(vector<int>& nums) {
        int n = nums.size();
        int actualSum = n*(n+1)/2;
        int arraySum = 0;
        for(int i = 0;i<nums.size();i+=1) {
            arraySum += nums[i];
        }
        return actualSum - arraySum ;
        }
};

File: 0268-missing-number/0268-missing-number.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var missingNumber = function(nums) {
    let unique = new Set(nums);

    for(let i =1;i<=nums.length;i++) {
        if(!unique.has(i)) {
            return i;
        }
    }
    return 0;
};